src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/alerts/Transformer.kt

```diff
@@ -0,0 +1,44 @@
+package uk.gov.justice.digital.hmpps.prisonerfromnomismigration.alerts
+
+import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.alerts.model.NomisAlert
+import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.nomissync.model.AlertResponse
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
+// The initial contract with DPS was to send the entire NOMIS alert, hence the DTO name. This is likely to change
+// but for now copy as if we are sending all this data
+@Suppress("ktlint:standard:discouraged-comment-location")
```

Comment:
I'm going to ask to see how we can remove this rule, not convinced it helps anyone

Reply:
I've read the docs and think the rule is correct. The problem is that this
```offenderNo = "A1234TT", // TODO - return in NOMIS API``` when read by a parser looks like the comment applies to the line below (because it comes after the `,`). Hence it's always better to put the comment above the line in a parameter list, e.g.
```
 // TODO - return in NOMIS API
offenderNo = "A1234TT",
```